src/server/AppManager.ts

```diff
@@ -380,7 +413,9 @@ export class AppManager {
         return true;
     }
 
-    public async add(appPackage: Buffer, enable = true, marketplaceInfo?: IMarketplaceInfo): Promise<AppFabricationFulfillment> {
+    public async add(appPackage: Buffer, installationParameters: IAppInstallParameters): Promise<AppFabricationFulfillment> {
+        const { enable, marketplaceInfo, permissionsGranted } = installationParameters;
```

Comment:
`enable` now does not have the default value `true`, making newly added apps to be disabled after installation. I'm changing this :grin: